scripts/postbuild.ts

```diff
@@ -0,0 +1,94 @@
+import { readFileSync, writeFileSync } from 'node:fs'
+import { resolve } from 'node:path'
+
+function patchCjs(cjsModulePath: string, name: string) {
+  const cjsModule = readFileSync(cjsModulePath, 'utf-8')
+  writeFileSync(
+    cjsModulePath,
+    cjsModule.replace(`module.exports = ${name};`, `exports.default = ${name};`),
+    { encoding: 'utf-8' },
+  )
+}
+
+function patchDCts(ctsModulePath: string, name: string) {
+  let content = readFileSync(ctsModulePath, 'utf-8').replace(`, ${name} as default `, ' ')
+  content += `\nexport = ${name};`
+  writeFileSync(
+    ctsModulePath,
+    content,
+    { encoding: 'utf-8' },
+  )
+}
+
+function patchTsupCjs(cjsModuleName: string, name: string) {
+  let file = resolve(`${cjsModuleName}.cjs`)
+  let content = readFileSync(file, 'utf-8')
+  writeFileSync(
+    file,
+    content.replace(`module.exports = __`, `exports.default = __`),
+    { encoding: 'utf-8' },
+  )
+  file = resolve(`${cjsModuleName}.d.cts`)
+  content = readFileSync(file, 'utf-8').replace(`, ${name} as default `, ' ')
+  content += `\nexport = ${name};`
+  writeFileSync(
+    file,
+    content,
+    { encoding: 'utf-8' },
+  )
+}
+
+// @unocss/astro
+patchCjs(resolve('./packages/astro/dist/index.cjs'), 'UnoCSSAstroIntegration')
```

Comment:
I don't feel this is a long term solution, as everytime we change the export we need to remeber to update here. And also there is no way to gurad this when it goes wrong. I guess we should have a more automated way.

Reply:
yeah, that was first approach, we can extract logic via regex, maybe we should fix this in unbuild

Reply:
this is only about the default export (in CJS), the rest of exports can be there